tests/Unit/DateServiceTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Tests\Unit;
+
+use App\Models\Booking;
+use App\Services\DateService;
+use PHPUnit\Framework\TestCase;
+
+class DateServiceTest extends TestCase
```

Comment:
These tests look great - what we need now though are some HTTP tests. You've got these unit tests covering your service, you should have a full set of HTTP tests that make sure the overall code does what it's supposed to. 

I actually missed this as part of the story 1 and 2 pull requests to be honest - the routes there require HTTP tests too.